src/scenes/Modals/ConfirmLeaveModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { BaseM } from 'components/core/Text/Text';
+import breakpoints from 'components/core/breakpoints';
+import Button from 'components/core/Button/Button';
+import Spacer from 'components/core/Spacer/Spacer';
+import { useModal } from 'contexts/modal/ModalContext';
+import { useCallback } from 'react';
+import styled from 'styled-components';
+// import { useAuthenticatedUsername } from 'hooks/api/users/useUser';
+import useBackButton from 'hooks/useBackButton';
+
+function ConfirmLeaveModal() {
+  const { hideModal } = useModal();
+
+  // FIXME: Do we want escape on /edit to go back to the previous state in history? Or simply always go to /{username}?
+  const username = 'connorr'; // useAuthenticatedUsername();
```

Comment:
The new graphQL work goes a bit over my head and I need some help identifying the best way to get the user's username using this new configuration (we have removed `useAuthenticatedUsername` since I started work on this

Reply:
haha foshooo let's schedule time to pair on this!